src/backend/distributed/utils/metadata_cache.c

```diff
@@ -2051,6 +2052,42 @@ CitusTextSendAsJsonbFunctionId(void)
 }
 
 
+/*
+ * PgTableVisibleFuncId returns oid of the pg_table_is_visible function.
```

Comment:
we usually put short function comments should be in single line like
```c
/* PgTableVisibleFuncId returns oid of the pg_table_is_visible function. */
```
However, I saw many exceptions to this in the code. I guess we are not very strict on this.